tests/test_ledger.py

```diff
@@ -232,3 +232,65 @@ def test_delete_non_existent_ledger():
     cashctrl_ledger = CashCtrlLedger()
     with pytest.raises(RequestException):
         cashctrl_ledger.delete_ledger_entry(ids='non-existent')
+
+def test_mirror_ledger(add_vat_code):
+    cashctrl_ledger = CashCtrlLedger()
+    target = StandaloneLedger.standardize_ledger(pd.concat([individual_transaction, collective_transaction]))
+    target['date'] = target['date'].ffill()
+    initial_ledger = cashctrl_ledger.ledger().reset_index(drop=True)
+
+    # Mirror with one single and one collective transaction
+    cashctrl_ledger.mirror_ledger(target=target)
+    mirrored = cashctrl_ledger.ledger().reset_index(drop=True)
+    m = target.merge(mirrored.drop(columns=['id']), how='left', indicator=True)
+    assert (m['_merge'] == 'both').all(), (
+            'Mirroring error: Some target accounts were not mirrored'
+        )
+
+    # Mirror with duplicate transactions and delete=False
+    target = StandaloneLedger.standardize_ledger(pd.concat([
+        individual_transaction,
+        individual_transaction,
+        collective_transaction,
+        collective_transaction
+    ]))
+    target['date'] = target['date'].ffill()
+    cashctrl_ledger.mirror_ledger(target=target)
+    mirrored = cashctrl_ledger.ledger().reset_index(drop=True)
+    m = target.merge(mirrored.drop(columns=['id']), how='left', indicator=True)
+    assert len(target) == len(mirrored)
+    assert (m['_merge'] == 'both').all(), (
+            'Mirroring error: Some target ledger were not mirrored'
+        )
+
+    # Mirror with delete=False
+    before_mirroring = cashctrl_ledger.ledger().reset_index(drop=True)
+    target = StandaloneLedger.standardize_ledger(pd.concat([individual_transaction, collective_transaction]))
+    target['date'] = target['date'].ffill()
+    cashctrl_ledger.mirror_ledger(target=target, delete=False)
+    mirrored = cashctrl_ledger.ledger().reset_index(drop=True)
+    m = before_mirroring.merge(mirrored.drop(columns=['id']), how='left', indicator=True)
+    assert len(before_mirroring) == len(mirrored)
+    assert (m['_merge'] == 'both').all(), (
+            'Mirroring error: Some target ledger were not mirrored'
+        )
+
+    # Mirror with delete=True
+    target = StandaloneLedger.standardize_ledger(pd.concat([individual_transaction, collective_transaction]))
+    target['date'] = target['date'].ffill()
+    cashctrl_ledger.mirror_ledger(target=target)
+    mirrored = cashctrl_ledger.ledger().reset_index(drop=True)
+    m = target.merge(mirrored.drop(columns=['id']), how='left', indicator=True)
+    assert len(target) == len(mirrored)
+    assert (m['_merge'] == 'both').all(), (
+            'Mirroring error: Some target ledger were not mirrored'
+        )
+
```

Comment:
Please add a test where target is empty and delete=True and confirm remote ledger is empty.